build-system-tests/templates/components/react/next/App.js

```diff
@@ -4,12 +4,14 @@ import {
   AccountSettings,
   Authenticator,
   FileUploader,
-  MapView,
   Text,
-  InAppMessagingProvider,
-  InAppMessageDisplay,
 } from '@aws-amplify/ui-react';
 import { StorageManager } from '@aws-amplify/ui-react-storage';
+import {
+  InAppMessageDisplay,
+  InAppMessagingProvider,
+} from '@aws-amplify/ui-react-notifications';
+import { MapView, LocationSearch } from '@aws-amplify/ui-react-geo';
 import '@aws-amplify/ui-react/styles.css';
```

Comment:
Can you include `import '@aws-amplify/ui-react-geo/styles.css'` after this line?
This has Geo related styles.
Same comment for other examples.

Reply:
Done! Thanks for catching!